From a9e5ed229f58a26189694276ce46d27ca8cdb0ff Mon Sep 17 00:00:00 2001
From: Bariccatti <bariccatti@gmail.com>
Date: Sat, 25 Apr 2026 13:31:46 -0300
Subject: [PATCH] Add AndroidManifest.xml for audiocontrol-caremu stub

The audiocontrol-caremu android_library module requires an AndroidManifest.xml. Add a minimal one.
---
 automotive_stubs/AndroidManifest.xml | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 automotive_stubs/AndroidManifest.xml

diff --git a/automotive_stubs/AndroidManifest.xml b/automotive_stubs/AndroidManifest.xml
new file mode 100644
index 0000000..a7eb636
--- /dev/null
+++ b/automotive_stubs/AndroidManifest.xml
@@ -0,0 +1,3 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="android.automotive.stubs">
+</manifest>
\ No newline at end of file
-- 
2.53.0.windows.3
